plugins/foreman_tasks/0.8/index.md

```diff
@@ -0,0 +1,302 @@
+
+    state = paused
+    state = running
+    state = stopped
+    
+    result = error
+    result = pending
+    result = warning
+    result = success
+    
+    state = stopped and result = failed
+
+as well as start and end time, to find tasks started after December 1st or finished after the same date, you can use following
+
+     start_at > 2016-12-01
+     ended_at > 2016-12-01 
+
+To get a quick overview of tasks status, you can use two dashboard widgets provided by the plugin.
+
+![Widget](/plugins/foreman_tasks/{{page.version}}/dashboard1.png)
+![Widget](/plugins/foreman_tasks/{{page.version}}/dashboard2.png)
+
+## 3.2 Task in detail
+
+Each task has following attributes
+
+
+- **Name** - human readable name of the task, it may differ per task of the same type since task inputs can be used for customization of the value 
+- **Result** - one of *success*, *error*, *warning*, *pending*, while success and error are probably clear, warning and pending is sometimes source of a confusion. *pending* means we can't tell yet since the task hasn't finished yet, *warning* means that some error occurred, user fixed the error and resumed the task that finished successfully afterwards. In other words, the warning means that someone had to do a manual step to finish the task. 
+- **State** - one of *running*, *stopped*, *paused*, running tasks are tasks that are actively being performed. If an error occurres during the run, based on the task rescue strategy, it can be paused or switched to stopped. Paused tasks can be resumed making them running again. After all task steps are finished the task state is stopped. Stopped tasks can't be resumed. 
+- **Triggered by** - user who started the task
+- **Execution type** - possible values are *immediate* and *delayed*, delayed tasks are usually scheduled to start at specific time in future, to determine the execution type, attributes *start_at* and *started_at* are compared
+
+- **Start at** - indicates the time when the task should start
+- **Started at** - indicates the time when the task really started
+- **End at** - indicates the time when the task stopped
+- **Start before** - indicates the deadline after which the task should not start if it hasn't started yet
+
+- **Progress** - the task may consist of multiple steps or sub-tasks, the progress is calculated based on percentage of finished steps/sub-tasks
+
+Note that you get more information by hovering some of the attribute values.
+
+### 3.2.1 Results and State related Q&A
+
+- Q: Why I see paused task?
+- A: Most common cause is that the task was programmed to pause on errors and some error occurred. On the other hand, there might be tasks that are paused because they are waiting to be resumed by some external service when it's ready. To find out more information, check if you see *Errors* field on task page or red box under the Errors tab. If you see something, the pause was very likely caused by an error.
```

Comment:
IMHO swap the condition, e.g.
an error has occurred and it was programmed to pause on error